addons/apecloud-mysql/templates/clusterversion-auditlog.yaml

```diff
@@ -23,7 +23,7 @@ spec:
               - |
                 cp /usr/lib/mysql/plugin/ {{ .Values.mysqlConfigs.dataMountPath }}/plugin -r
             imagePullPolicy: Always
-            image: {{ .Values.image.registry | default "docker.io" }}/apecloud/mysql_audit_log:8.0.33
+            image: {{ .Values.image.registry | default "docker.io" }}/mysql_audit_log:8.0.33
```

Comment:
why remove 'apecloud' repo ? if we set image.registry to our aliyun registry, the link is invalid.